testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -411,7 +412,7 @@ private void testCompression(boolean optimizedWriter, String compressionCodec)
                 assertThat(onTrino().executeQuery("SELECT * FROM " + trinoTableName))
                         .containsOnly(expected);
 
-                if ("ZSTD".equals(compressionCodec) && !databricksRuntimeVersion.equals(DATABRICKS_104_RUNTIME_VERSION)) {
+                if ("ZSTD".equals(compressionCodec) && databricksRuntimeVersion.compareTo(DATABRICKS_104_RUNTIME_VERSION) < 0) {
```

Comment:
I missed that. Does Databricks 10.4 add ZSTD support out of the box?
🎉  finally we can switch Delta's default compression to ZSTD, right?

cc @losipiuk @alexjo2144